lib/sessions.ts

```diff
@@ -85,6 +109,98 @@ export class Sessions {
     });
   }
 
+  /** Parse a JWT and verify the signature, preferring local verification over remote.
+   *
+   * If maxTokenAge is set, remote verification will be forced if the JWT was issued at (based on
+   * the "iat" claim) more than maxTokenAge seconds ago.
+   */
+  async authenticate_jwt(
```

Comment:
Should we have a force no matter what method as well? Perhaps for a sensitive action you'd want to be 100% sure the session is still valid (user did something on another platform)

Reply:
I suppose you could go back to the opaque session token for that (assuming the developer decided to hold onto it)

Reply:
> Should we have a "force remote verification no matter what" method as well? Perhaps for a sensitive action you'd want to be 100% sure the session is still valid (user did something on another platform)

We do! In this method, you can set `maxTokenAge: 0`. Or you can call the `authenticate` method (always uses the network) with `{session_jwt: jwt}`. 

I'll update the doc comment here to add that usage note, but I'm happy to keep discussing usability for that flow.